tf-modules/cumulus/data_migration.tf

```diff
@@ -0,0 +1,19 @@
+module "data_migration_lambda" {
```

Comment:
You can probably ignore this file as it's likely to change pending the outcome of the conversation about how to run the data migration